src/test/playwright-fixture.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+import { Configuration, FrontendApi, IdentityApi } from "@ory/client"
+import { test as base } from "@playwright/test"
+
+type TestFixtures = {
+  sdk: FrontendApi
+  adminSdk: IdentityApi
+  environment: {
+    applicationUrl: string
+    oryProjectUrl: string
+    projectApiToken: string
+  }
+}
+
+export const test = base.extend<TestFixtures>({
+  environment: [
+    // eslint-disable-next-line no-empty-pattern
+    async ({}, use) => {
+      const environment = {
+        // applicationUrl is the URL where the application is running
+        // this should be the custom domain where these tests will be run on
+        // e.g. https://app.example.com
+        applicationUrl: process.env.APPLICATION_URL || "http://localhost:3000",
+        // oryProjectUrl is the Ory Network API URL
+        // this should be the custom domain where these tests will be run on
+        // e.g. https://auth.example.com which points to https://<slug>.projects.oryapis.com
+        // use the ory cli tunnel url instead if you are running tests locally without a custom domain.
+        oryProjectUrl: process.env.ORY_PROJECT_URL || "http://localhost:4000",
+        // projectApiToken is the Ory Network API token
+        // this should only be used when running tests that alter the state of the Ory project
+        // for example when creating a new user
+        // and deleting it after the test has run
+        projectApiToken: process.env.ORY_PROJECT_API_TOKEN || "",
+      }
+      await use(environment)
+    },
+    { auto: false },
+  ],
+  sdk: [
+    async ({ environment }, use) => {
+      const sdk = new FrontendApi(
+        new Configuration({
+          basePath: environment.oryProjectUrl,
+        }),
+      )
```

Comment:
This is using the default axios SDK, right? Axios is annoying to debug in playwright, as its failure traces are not connected to playwright traces. The solution is, to use playwright's `request` module instead. 

Since you're already using a fixture here, you can just create an axios adapter that "delegates" the request to playwright. I have already done that [here](https://github.com/ory-corp/cloud/blob/master/test/e2epw/src/tests/sdk/identity.spec.ts#L47-L79). 

Doing that, will help you (or anyone else) debug why network requests are failing, etc.